pkg/solana/relay.go

```diff
@@ -22,9 +22,7 @@ type TransmissionSigner interface {
 }
 
 // TODO: Goes away with solana txm
-type KeyStore interface {
-	Get(id string) (solkey.Key, error)
-}
+type GetKey func(id string) (TransmissionSigner, error)
```

Comment:
This works around the `package solkey` import. The messy part is contained in core, fortunately.
https://github.com/smartcontractkit/chainlink/pull/6629/files#r874192985

Reply:
Consider renaming the `TransmissionSigner` interface simply as `Signer` and this type as:

```suggestion
type SignerProvider func(id string) (Signer, error)
```

Reply:
Much better!